EIPS/eip-7092.md

```diff
@@ -0,0 +1,342 @@
+    * @param _amount the amount of bonds to be transferred
+    * @param _data additional information provided by the token holder
+    */
+    event Transferred(address _from, address _to, uint256 _amount, bytes _data);
+
+    /**
+    * @notice MUST be emitted when an account is approved
+    * @param _owner the bonds owner
+    * @param _spender the account to be allowed to spend bonds
+    * @param _amount the amount allowed by _owner to be spent by _spender.
+    */
+    event Approved(address _owner, address _spender, uint256 _amount);
+
+    /**
+    * @notice MUST be emmitted when the `_owner` decreases allowance from `_sepnder` by quantity `_amount`
+    * @param _owner the bonds owner
+    * @param _spender the account that has been allowed to spend bonds
+    * @param _amount the amount of tokens to disapprove
+    */
+    event AllowanceDecreased(address _owner, address _spender, uint256 _amount);
+}
+```
+
+## Rationale
+
+The financial bond standard is designed to represent fixed income assets, which reprensent a loan made by an investor to a borrower.
+The proposed design has been motivated by the necessity to tokenize fixed income assets, and to represent the bond token with same
+characteristics as in traditional finance. Keeping the same properties as in tradional finance is necessary for issuers and investors
+to move to tokenized bonds without major difficulties. The same structure used in tradional finace, i.e issuer-investment bank-investors
+can be used for the bond standard, in that case the investment bank intermediary may be replaced by smart contracts. In the case of
+institutional issuance, the smart contracts can be managed by the investment bank. Decentralized exchanges may also use the bond standard
+to list bonds, in that case, decentralized exchanges will be in charge of managing the smart contracts. Other entities may also create
+tokenized bonds by integrating this financial bond interface.
+
+Tokenizing bonds will offer several advantages compared to traditional bond issuance and trading, among with:
+
+1. Fractional ownership: The bond standard does not limit the bond denomination to some minimum value compared to traditioanal bonds where the denomination is typically equal to $100 or $1,000.
+2. Accessibility: By allowing lower investment thresholds, tokenized bonds are supposed to attract retail investors who could not participate in traditional markets due to high minimum investment requirements.
+3. Increased liquidity: Fractioanal ownership will bring new investors in the bond market, this will increase liquidity in the bond market.
+4. Cost savings: By replacing intermediaries with smart contracts, bond's tokenization will reduce costs associated with the bond issuance and management.
+5. Easy accessibility and 24/7 trading: Tokenized bonds are supposed to be traded on digital platforms such as decentralized exchanges. Therefore, they will be more accessible compared to tradional bond market.
```

Comment:
This list belongs in motivation. The rationale section should describe technical choices made within the EIP itself, while the motivation should justify the existence of the EIP as a whole.

A good example of what you _should_ include in your rationale section is why you don't extend an existing token standard, like ERC-20 or ERC-1155.